tests/unit/test-cmd/test.lint.js

```diff
@@ -7,9 +7,13 @@ import defaultLintCommand from '../../../src/cmd/lint';
 import {FileFilter} from '../../../src/cmd/build';
 import {fake, makeSureItFails} from '../helpers';
 
+type setUpParams = {
+  createLinter?: Function,
+  fileFilter?: Function,
+}
```

Comment:
nit: this should have a blank line after it, separating this block from the call to `describe()`